doc/machines/index.rst

```diff
@@ -38,28 +40,47 @@ Parameters:
 
 ``user`` (optional, default=root)
 
+``service_id`` (required)
+
 When the SSH token type is assigned to a client, the user specified in the
 user parameter
 can login with the private key of the SSH token.
 
-In the ``sshd_config`` file you need to configure the ``AuthorizedKeysCommand``.
-Set it to::
+The ``service_id`` identifies the SSH servers or group of SSH servers, where the login is allowed to occur.
+
+To facilitate this, the SSH server fetches the managed SSH keys from the privacyIDEA server on demand.
+The SSH server uses the ``AuthorizedKeysCommand`` in the ``sshd_config`` to do this.
+
+There is an Python script `privacyidea-authorizedkey` in the privacyideaadm repository. Note, that this
```

Comment:
Maybe we should add a link to the privacyideaadm repository to avoid confusion with the part about the shell script (since they are both called `privacyidea-authorizedkey(s)`.
Do we need to support both scripts? Should we streamline the documentation for SSH-keys to one dedicated workflow?

Reply:
Does it hinder this PR from being merged?